stories/hooks/useThing.stories.mdx

```diff
@@ -0,0 +1,24 @@
+import { Meta } from "@storybook/addon-docs/blocks";
+
+<Meta title="Hooks/useThing" />
+
+# useThing
+
+A hook to access the nearest Thing from context if no arguments are supplied, or to request a Thing from given iris.
+
+Should be located within a `SessionProvider` if using to fetch.
+
+```
+const { thing, error } = useThing(datasetIri, thingIri, options)
+```
+
+## Parameters
+
+- `datasetIri`: (optional) a dataset iri to fetch the Thing from, if both `datasetIri` and `thingIri` are supplied. Uses the Dataset from context if omitted.
+- `thingIri`: (optional) a thing iri to fetch the Thing from. Uses the Thing from context if omitted.
+- `options`: (optional) additional options to pass to the dataset request, if `datasetIri` is supplied
```

Comment:
what are these options?

Reply:
the options here: https://docs.inrupt.com/developer-tools/api/javascript/solid-client/modules/_resource_soliddataset_.html#getsoliddataset

The only one actually documented there is `fetch`, which is the one useDataset/useThing deal with automatically

Reply:
@nicolasmondada I've just added links to those getsoliddataset docs